localization/english/culture/wc_basic_innovations_l_english.yml

```
@@ -0,0 +1,27 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ innovation_animal_war_training:0 "Animal War Training"
+ innovation_animal_war_training_desc:0 "Through skills taught to us my the best beastmasters, we can communicate, train and discipline the animals around us."
+ 
+ innovation_rogues_guild:0 "Rogues Guild"
+ innovation_rogues_guild_desc:0 "."
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+ innovation_rogues_school:0 "School of the Rogue"
```

I'd like to see a different title here. School of the Rogue sounds like it is referring to an individual rogue, rather than for rogues.

Ideas: Rogue Training Academy, School of Roguery, Assassin Training Camp, Rogue Academy.
```suggestion
 innovation_rogues_school:0 "School of Roguery"
```